.github/workflows/ci_tests.yaml

```diff
@@ -65,10 +65,11 @@ jobs:
 
       # Setup Miniconda
       - name: Setup Miniconda
-        uses: goanpeca/setup-miniconda@v1.6.0
+        uses: conda-incubator/setup-miniconda@v1.7.0
         with:
           python-version: ${{ matrix.python-version }}
           channels: conda-forge
+          miniconda-version: "latest"
```

Comment:
I think you'll like this @seisman, The Windows conda install speed up (~24min -> ~8min) comes from just adding this one line! What it does (on Windows) is to update the miniconda version from 4.6.14 to 4.8.3, and this would use a newer/faster `conda` solver, among other things.

Reply:
It also speed up macOS. Perhaps we should also update `ci_tests_dev.yaml`?

Reply:
Not really by much, I think we can do it another time since that’s not really a bottleneck.